test/JDBC/upgrade/13_6/schedule

```diff
@@ -301,4 +301,4 @@ sys-sql_modules_before-14_7-or-15_2
 sys-all_sql_modules_before-14_5
 BABEL_OBJECT_ID
 test_windows_login_before_15_2
-BABEL_OBJECT_NAME
\ No newline at end of file
+BABEL_OBJECT_NAME
```

Comment:
why do you have theses changes, new and old look the same to me

Reply:
This must have happened while resolving merge conflicts. It should not cause any issue.